### PR TITLE
New Data Source: `azurerm_private_dns_resolver_inbound_endpoint`

### PR DESCRIPTION
**Pull request by pchanvallon**
``` txt
=== RUN   TestAccDNSResolverInboundEndpointDataSource_basic
=== PAUSE TestAccDNSResolverInboundEndpointDataSource_basic
=== CONT  TestAccDNSResolverInboundEndpointDataSource_basic
--- PASS: TestAccDNSResolverInboundEndpointDataSource_basic (434.96s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/privatednsresolver    436.195s

=== RUN   TestAccDNSResolverInboundEndpointDataSource_complete
=== PAUSE TestAccDNSResolverInboundEndpointDataSource_complete
=== CONT  TestAccDNSResolverInboundEndpointDataSource_complete
--- PASS: TestAccDNSResolverInboundEndpointDataSource_complete (604.47s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/privatednsresolver    605.711s
```

### CODE REVIEW

**Comment by pchanvallon** [comment]
Following PR #19947 review, I have implemented the same suggestions. 

Here is the updated test result :
``` txt
=== RUN   TestAccDNSResolverInboundEndpointDataSource_basic
=== PAUSE TestAccDNSResolverInboundEndpointDataSource_basic
=== CONT  TestAccDNSResolverInboundEndpointDataSource_basic
--- PASS: TestAccDNSResolverInboundEndpointDataSource_basic (435.39s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/privatednsresolver    436.573s
```

File: website/docs/d/private_dns_resolver_inbound_endpoint.html.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+* `subnet_id` - The subnet ID of the IP configuration.
+
+* `private_ip_address` - The private IP address of the IP configuration.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/language/resources/syntax#operation-timeouts) for certain actions:
+
+* `read` - (Defaults to 5 minutes) Used when retrieving the Private DNS SRV Record.
```
Comment (mbfrahry) [inline website/docs/d/private_dns_resolver_inbound_endpoint.html.markdown:56]:
```suggestion
* `read` - (Defaults to 5 minutes) Used when retrieving the Private DNS Resolver Inbound Endpoint.
```

File: internal/services/privatednsresolver/private_dns_resolver_inbound_endpoint_data_source_test.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	ip_configurations {
+	  subnet_id = azurerm_subnet.test.id
+	}
+  }
+
+  data "azurerm_private_dns_resolver_inbound_endpoint" "test" {
+	  name         		    = azurerm_private_dns_resolver_inbound_endpoint.test.name
+	  private_dns_resolver_id = azurerm_private_dns_resolver.test.id
+  }
+`, data.Locations.Primary, data.RandomInteger)
```
Comment (mbfrahry) [inline internal/services/privatednsresolver/private_dns_resolver_inbound_endpoint_data_source_test.go:85]: Same comment as before

**Review by mbfrahry — CHANGES_REQUESTED** [review]
This looks good as well with just similar comments as the other PRs.

File: website/docs/d/private_dns_resolver_inbound_endpoint.html.markdown (hunk 1 above)
Comment (pchanvallon) [reply website/docs/d/private_dns_resolver_inbound_endpoint.html.markdown]: done

File: internal/services/privatednsresolver/private_dns_resolver_inbound_endpoint_data_source_test.go (hunk 2 above)
Comment (pchanvallon) [reply internal/services/privatednsresolver/private_dns_resolver_inbound_endpoint_data_source_test.go]: done

**Comment by pchanvallon** [comment]
Thanks again for you review, I've updated the documentation and tests following your comments.

**Pull request merged**